packages/core/src/common/classes.ts

```diff
@@ -163,6 +163,8 @@ export const INPUT_GROUP = `${INPUT}-group`;
 export const INPUT_LEFT_CONTAINER = `${INPUT}-left-container`;
 export const INPUT_ACTION = `${INPUT}-action`;
 
+export const INPUT_RESIZABLE_INPUT_SPAN = `${NS}-resizable-input-span`;
```

Comment:
nit: the variable name should match the class name more closely:

```suggestion
export const RESIZABLE_INPUT_SPAN = `${NS}-resizable-input-span`;
```